src/Asm89/Stack/CorsService.php

```diff
@@ -37,140 +37,104 @@ private function normalizeOptions(array $options = array())
         );
 
         // normalize array('*') to true
-        if (in_array('*', $options['allowedOrigins'])) {
+        if ($options['allowedOrigins'] === array('*')) {
```

Comment:
This could be breaking if you have ['*','something.com']

Reply:
Fixed!